source/installation/install-on-an-unattended-server/rhel-centos-server-installation/rhel-centos-server-installation.txt

```diff
@@ -1,12 +1,30 @@
+.. _rhel-install:
+
 ================================
 RHEL/CentOS Systems Installation
 ================================
 
-TBD.
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Install Relational Migrator on an unattended server in a Linux systems. For instructions 
+on installation and configuration, see the following: 
```

Comment:
(suggestion)
Same as Windows. Consider closing with "see the installation tasks."

Reply:
Done.